scripts/select_appointments.sql

```diff
@@ -1 +1,10 @@
 -- retrieve appointment information
+
+-- Using past repo but haven't had time to update with appropriate data.
+SELECT COUNT(*), d.appointments
+FROM patients p
+INNER JOIN doctors d
+  ON p.doctor_id = d.id
+WHERE d.specialty <> 'General practice'
+GROUP BY d.specialty
+ORDER BY COUNT(*) ASC;
```

Comment:
Not quite. Recall what the question was asking: "Write a query to retrieve the names and appointment times for everyone seeing Dr. Hubbard." Will `COUNT` give us the type of data we are looking for? 

```sql
-- for Dr. Hubbard
SELECT p.surname, a.date, a.time
  FROM patients p
  INNER JOIN appointments a ON a.patient_id = p.id
  INNER JOIN doctors d ON a.doctor_id = d.id
    WHERE d.surname = 'Hubbard';

-- for patient Wilcox
SELECT d.surname, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a ON a.doctor_id = d.id
  INNER JOIN patients p ON a.patient_id = p.id
    WHERE p.surname = 'Wilcox';
```